flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/batch/table/SetOperatorsITCase.scala

```diff
@@ -225,4 +227,62 @@ class SetOperatorsITCase(
     val expected = "2,1,Hi\n" + "3,2,Hello\n" + "4,2,Hello world\n"
     TestBaseUtils.compareResultAsText(results.asJava, expected)
   }
+
+  @Test
+  def testInWithFilter(): Unit = {
```

Comment:
Add `stringexpr` tests to validate equivalence of Scala expression and Java String Table API.